package.json

```diff
@@ -56,14 +57,16 @@
     "lint": "NODE_ENV=dev ./node_modules/.bin/eslint src && ./node_modules/.bin/stylelint 'src/**/*.scss'",
     "npm": "OUTPUT=dist LANGUAGE=en-US REACT=true yarn run build-prod",
     "nsp": "nsp check --reporter summary",
-    "major": "./build/release.sh -m && ./build/publish.sh",
-    "minor": "./build/release.sh -n && ./build/publish.sh",
-    "patch": "./build/release.sh -p && ./build/publish.sh",
+    "major": "./build/release.sh -m && ./build/publish.sh && ./build/publish-examples.sh",
```

Comment:
Minor: Should `publish-examples.sh` be called from `publish.sh`, instead? It'll help ensure that the examples are always kept updated, even if `publish.sh` is called out of band (like by CI, or something).